modules/learn/pages/data/scopes-and-collections.adoc

```diff
@@ -8,14 +8,28 @@ Couchbase Server provides _scopes_ and _collections_; allowing documents to be c
 
 A _collection_ is a data container, defined on Couchbase Server, within a bucket whose type is either _Couchbase_ or _Ephemeral_.
 Up to 1000 collections can be created per cluster.
-Bucket-items can optionally be assigned to different collections according to content-type.
+Item-names must be unique within their collection.
+Items can optionally be assigned to different collections according to content-type.
 For example, within a bucket that contains travel information, documents that relate specifically to _airports_ might be assigned to an _airports_ collection, while documents that relate to _hotels_ might be assigned to a _hotels_ collection, and so on.
+Applications can be assigned per-collection access-rights; allowing each application to access only those collections it requires.
+
+A collection can be created and managed by means of the Couchbase Web Console UI, the Couchbase CLI, the Couchbase REST API, and N1QL.
+A collection can be indexed; and it can be dropped.
+The data in a collection can be replicated, by means of _Cross Data Center Replication_ (XDCR), to another collection in the same or in a different bucket.
 
 A _scope_ is a mechanism for the grouping of multiple collections.
 Up to 1000 scopes can be created per cluster.
 Collection-names must be unique within their scope.
-Collections might be assigned to different scopes according to content-type, or to deployment-phase (ie, test versus production). 
-Applications can be assigned per-scope access-rights; allowing each application to access only those collections it requires.
+Collections might be assigned to different scopes according to content-type, or to deployment-phase (ie, test versus production).
+Applications can be assigned per-scope access-rights; allowing each application to access only those scopes it requires.
+
+A scope can be created and managed by means of the Couchbase Web Console UI, the Couchbase CLI, the Couchbase REST API, and N1QL.
```

Comment:
same I think - check with SDK team if it's worth a statement here

Reply:
Thanks. Have done so. Will include a pointer to their developer preview docs.